WEBPAGE/main.py

```
@@ -0,0 +1,911 @@
+    return True
+
+def send_email_code(to_email, code):
+    to_email = os.environ.get('email') # for testing
+    try:
+        with smtplib.SMTP("smtp.gmail.com", 587) as server:
+            server.starttls()
+            server.login(os.environ.get('email'), os.environ.get('app_password'))
+            message = f"Subject: Your MFA Code\n\nYour verification code is: {code}"
+            server.sendmail(os.environ.get('email'), to_email, message)
+        logger.info(f"Email sent to {to_email}")
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)